src/crkeng/app/integration_tests/test_paradigm_manager.py

```diff
@@ -8,7 +8,17 @@
 from more_itertools import first, ilen
 
 from CreeDictionary.CreeDictionary.paradigm.generation import default_paradigm_manager
-from CreeDictionary.CreeDictionary.paradigm.manager import ParadigmManager
+from CreeDictionary.CreeDictionary.paradigm.manager import (
+    ParadigmManager,
+    ParadigmManagerWithExplicitSizes,
+)
+
+
+def test_paradigm_sizes_are_ordered(paradigm_manager):
+    assert isinstance(paradigm_manager, ParadigmManagerWithExplicitSizes)
+    assert (
+        paradigm_manager.all_sizes_fully_specified()
+    ), "please check MORPHODICT_PARADIGM_SIZES or the names of the .tsv files"
```

Comment:
This is intended to fail the CI if files in `res/layouts` are out of place, or if `settings.PARADIGM_SIZES` is misconfigured.